.github/workflows/pull.yml

```diff
@@ -0,0 +1,55 @@
+name: Pull actions
```

Comment:
I pushed this file to this PR by mistake. It is present in the original dev container but i don't know if it is needed or not.